OpenEdXMobile/src/main/java/org/edx/mobile/view/MyCoursesListFragment.java

```diff
@@ -72,6 +73,8 @@
 
     private FullScreenErrorNotification errorNotification;
 
+    private DialogErrorNotification dialogErrorNotification;
```

Comment:
This can be defined as local variable.

Reply:
If we declare a local variable then a new instance will be created each time a dialog has to be shown. Right now only one instance is created and the same instance is reused for all further notifications.